x/dex/types/keys.go

```diff
@@ -40,8 +40,8 @@ var (
 
 	// TokenPairKey is the store key for token pair
 	TokenPairKey = []byte{0x01}
-	// TokenPairNumberKey is the store key for token pair num
-	TokenPairNumberKey = []byte{0x02}
+	// TokenPairMaxIDKey is the store key for token pair max ID
+	TokenPairMaxIDKey = []byte{0x02}
```

Comment:
I think MaxTokenPairID is better than TokenPairMaxID, and should keep naming consistent:
TokenPairMaxIDKey -> MaxTokenPairIDKey